jobs/JEVS_MESOSCALE_PLOTS

```diff
@@ -29,7 +28,7 @@ export MET_PLUS_PATH=${MET_PLUS_PATH:-$METPLUS_PATH}
 export MET_PATH=$MET_ROOT
 export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
 export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
-export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${model_ver}}
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${model_ver}}
```

Comment:
@RoshanShrestha-NOAA Please change this line from
export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${model_ver}}
to 
export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${evs_ver}}